core/src/main/resources/defaults/blueprints/hdf31-flow-management.bp

```diff
@@ -0,0 +1,92 @@
+{
+  "inputs": [],
+  "description": "Useful for data-flow management with Apache NiFi",
+  "blueprint": {
+    "Blueprints": {
+      "blueprint_name": "hdf31-flow-management",
+      "stack_name": "HDF",
+      "stack_version": "3.1"
+    },
+    "configurations": [
+      {
+        "nifi-ambari-config": {
+          "nifi.security.encrypt.configuration.password": "changemeplease",
+          "nifi.node.protocol.port": "9089",
+          "nifi.node.port": "9090",
+          "nifi.node.ssl.port": "9091",
+          "nifi.max_mem": "1g"
+        }
+      },
+      {
+        "nifi-properties": {
+          "nifi.sensitive.props.key": "changemeplease",
+          "nifi.security.user.login.identity.provider": "kerberos-provider",
```

Comment:
If i'm not mistaken this forces us to use Kerberos for the BP. Was this intentional?

Reply:
Yes, usage of Kerberos is required at this point, rather than authenticating users with client certs, since that would take more interaction with the user to set up a cluster and begin using it.  Once Knox support (after the release of HDF 3.2) is available, we can look into removing the Kerberos requirement.